virttest/libvirt_xml/devices/watchdog.py

```diff
@@ -41,7 +41,7 @@ def try_modprobe(self, session):
         """
         handled_types = {"ib700": "ib700wdt", "diag288": "diag288_wdt"}
         if self.model_type not in handled_types.keys():
-            return False
```

Comment:
Please, add a `logging.debug`, something like `"Unknown model type. Skipping watchdog modeprobe for %s." % self.model_type`.

Reply:
Done